### ISSUE TITLE
Native Pony Inpainting

### ISSUE THREAD

**Issue by xx88xx** [lllyasviel/Fooocus#3364]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do?

 As I understand it, the inpainting module (e.g., v2.6) in Fooocus isn’t trained for Pony, so it doesn’t work very well, if at all.

Would it be possible to have a dedicated Pony module (trained on Pony)?

Inpainting is one of the strongest features and main reasons to use Fooocus. It would be amazing to have this feature!

### Proposed workflow

When using the pony preset, switch the Inpainting module from the default to a pony Inpainting module.

**Comment by mashb1t** [comment]
@lllyasviel has not yet provided any information about the training of the model in https://github.com/lllyasviel/Fooocus/issues/1967. If you read this, it would be great if you could do so.

**Comment by kalle07** [comment]
i have same wishes...


at the moment i use an inpaint-pony model in webforgeUI
https://civitai.com/models/245423?modelVersionId=664247

you can use all sliders and options like before in SD15
but yes the inpaint is a bit noisy

**Comment by pureguttercleaning** [comment]
no i dont have information about your problem. i have solution about cleaing. We cater to all types of properties, including one-story, two-story, and three-story homes, pool houses, business complexes, industrial buildings, and more. Get in touch with us to discuss your specific needs and get [services](https://pureguttercleaningrochester.com/) tailored to your property.

**Comment by kalle07** [comment]
iv found out that in its strong depend on model
but
SAMPLER: "ddpm"  seems workling well

Scheduler: you can play, exponential(more details), lcm(more smooth) ... maybe some other better for you

**Comment by Eagle07-Sudo** [comment]
It seems that creating a model for **Flux** is now a priority in case there is an intention to create a new inpainting model.

**Comment by xx88xx** [comment]
> It seems that creating a model for **Flux** is now a priority in case there is an intention to create a new inpainting model.

I agree probably all possible resources should be put into adding flux. Seems the most robust way to move forward

**Comment by mohamed-benali** [comment]
For Pony you can use this settings on the Developer Debug Mode -> Inpaint
![imagen](https://github.com/user-attachments/assets/ebaef114-1e41-444c-9cb8-daaf04354e5d)

It works well for me including LORAs

**Comment by JedediahJ** [comment]
> For Pony you can use this settings on the Developer Debug Mode -> Inpaint ![imagen](https://github.com/user-attachments/assets/ebaef114-1e41-444c-9cb8-daaf04354e5d)
> 
> It works well for me including LORAs

I just made a account cause I wanted to say thank you to this reply. I did this and I'm able to inpaint with successful results with a Pony model! just be mindful of what's in your prompts.
